src/pages/manage-users/[id].tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import type { InferGetServerSidePropsType, GetServerSideProps } from "next";
+import { getAttendees, getEventById } from "@/lib/firebase/firestore";
+import AttendeeList from "@/components/AttendeeList";
+import { Attendee } from "@/interfaces/UserInterface";
+
+function ManageUsers({
+  attendees,
+  name,
```

Comment:
What does this prop refer to? The name is confusing, does it refer to the event name or to the attendee name? Change it accordingly.